docs/source/release/v6.9.0/SANS/New_features/35869.rst

```diff
@@ -0,0 +1,2 @@
+- Switching between a custom output file and an automatically-generated one can now be performed using the
```

Comment:
Is it worth referring to the ISIS SANS interface in this release note, just to be clear about exactly where in Mantid the change is?